rasa_sdk/interfaces.py

```diff
@@ -271,6 +272,29 @@ def add_slots(self, slots: List[EventType]) -> None:
             self.slots[event["name"]] = event["value"]
             self.events.append(event)
 
+    def get_intent_if_not_fallback(self) -> Optional[Text]:
```

Comment:
I think there is no need to issue `warnings` as users don't know beforehand which intent they are expecting. Having no `nlu_fallback` intent present is hence neither unexpected state nor wrong usage. What do you think?